docs/usage/PRODUCT_SYNC.md

```diff
@@ -84,7 +84,7 @@ events.
 
 - `beforeUpdateCallback`
 a filter function which can be applied on a generated list of update actions. It allows the user to intercept product 
-update and modify (add/remove) update actions just before they are send to CTP API.
+update and modify (add/remove) update actions just before they are sent to CTP API.
```

Comment:
```suggestion
**_update_** actions just before they are sent to CTP API.
```

Reply:
also in the `beforeCreateCallback` section: 
user to intercept product create requests modify the draft before the create request is sent to CTP API.

->

user to intercept product **_create_** requests modify the draft before the create request is sent to CTP API.


and in the rest CATEGORY_SYNC.md, INVENTORY_SYNC.md and PRODUCT_TYPE_SYNC.md

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/1a11827db898fe7f14c597699010b712f55d1b10

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/90c7178c9ffdfe0eb0a1378c5cbe4bd1c7d188cf

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/e328c7f92531b07561a117ba20833fdd52c154e2